From 30f910960f2c71325b14489ad3fb6fff6b9b97e5 Mon Sep 17 00:00:00 2001
From: Lu Jialin <lujialin4@huawei.com>
Date: Thu, 4 May 2023 11:33:08 +0800
Subject: [PATCH 1577/1646] [Huawei] psi: disable CONFIG_PSI_DEFAULT_DISABLED
 in openeuler_defconfig

Offering: HULK
HULK inclusion
category: feature
bugzilla: 188663

-------------------------------

disable CONFIG_PSI_DEFAULT_DISABLED in openeuler_defconfig

Signed-off-by: Lu Jialin <lujialin4@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 arch/x86/configs/openeuler_defconfig   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 57a659b34479..3a9d59fb8326 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -90,7 +90,7 @@ CONFIG_TASK_DELAY_ACCT=y
 CONFIG_TASK_XACCT=y
 CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
-CONFIG_PSI_DEFAULT_DISABLED=y
+#CONFIG_PSI_DEFAULT_DISABLED is not set
 CONFIG_PSI_CGROUP_V1=y
 
 # end of CPU/Task time and stats accounting
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index ce2369ed2e8f..b1ed7f628a7d 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -106,7 +106,7 @@ CONFIG_TASK_DELAY_ACCT=y
 CONFIG_TASK_XACCT=y
 CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
-CONFIG_PSI_DEFAULT_DISABLED=y
+#CONFIG_PSI_DEFAULT_DISABLED is not set
 CONFIG_PSI_CGROUP_V1=y
 # end of CPU/Task time and stats accounting
 
-- 
2.17.1

